css/styles.css

```diff
@@ -113,6 +116,82 @@ h1 {
   background-size: cover;
 }
 
+.save-to-favorites {
+  background-color: #ffa500;
+  font-size: 11px;
+  font-family: 'Anton', sans-serif;
+  color: white;
+  border: none;
+  width: 118px;
+  height: 33px;
+  margin-left: 50px;
+  border-radius: 5px;
+}
+
+.view-full-recipe {
+  background-color: #ffa500;
+  font-size: 11px;
+  font-family: 'Anton', sans-serif;
+  color: white;
+  border: none;
+  width: 118px;
+  height: 33px;
+  margin-right: 100px;
+  border-radius: 5px;
+}
+
+.list-holder {
+  line-height: 3;
+  margin-left: 336px;
```

Comment:
This is a lot of margin. Why is this necessary? Are you trying to center this element in the page?

Reply:
It's beacause they are not centered in the wireframes, so I am trying to adjust them to have the same identical as possible